src/pages/LocalScript.tsx

```diff
@@ -435,7 +436,9 @@ export function LocalScript({
         return;
       }
       const code = scriptInfo.data;
-      setResult({ type: "running" });
+      var lastStatus: RunResult = { type: "preparing", uiMsg: "Preparing job" };
```

Comment:
Proposed message `Preparing job` - After user clicks `Run` and before any DAAS requests are made.